rust/feature-flags/src/property_matching.rs

```diff
@@ -193,6 +196,12 @@ pub fn match_property(
             //     Ok(false)
             // }
         }
+        OperatorType::In | OperatorType::NotIn => {
+            // TODO: we handle these in cohort matching, so we can just return false here
+            // because by the time we match properties, we've already decomposed the cohort
+            // filter into multiple property filters
+            Ok(false)
+        }
```

Comment:
This seems safe enough to do – I handle these condition within `evaluate_cohort_filters`, not with `match_property` directly.